db/crud.go

```diff
@@ -2589,11 +2618,14 @@ func (db *DatabaseCollectionWithUser) CheckProposedRev(ctx context.Context, doci
 const (
 	xattrMacroCas         = "cas"
 	xattrMacroValueCrc32c = "value_crc32c"
+	versionVectorMacro    = "._vv."
+	xattrMacroVersion     = "vrs"
```

Comment:
Assuming this is just a temporary path (until we move to _vv) - otherwise I think I'd recommend a single constant for `_vv.vrs`.  Even so I'd remove the `.` delimiters from versionVectorMacro and just insert them when building the path (in xattrVersionPath).